### PR TITLE
Refactor ILocalFunctionStatement.LocalFunctionSymbol to .Symbol, upda…

### PR DESCRIPTION
**Pull request by 333fred**
…te doc comment on Body. Fixes https://github.com/dotnet/roslyn/issues/21287. Tagging @dotnet/analyzer-ioperation @dotnet/roslyn-compiler for review.

### CODE REVIEW

**Comment by 333fred** [comment]
@dotnet-bot retest windows_release_unit32_prtest please

**Comment by 333fred** [comment]
Test failure is https://github.com/dotnet/roslyn/issues/21588

File: src/Compilers/Core/Portable/Operations/ILocalFunctionStatement.cs [hunk 1]
```diff
@@ -16,10 +16,14 @@ public interface ILocalFunctionStatement : IOperation
...
         /// Local function symbol.
         /// </summary>
-        IMethodSymbol LocalFunctionSymbol { get; }
+        IMethodSymbol Symbol { get; }
         /// <summary>
         /// Body of the local function.
         /// </summary>
+        /// <remarks>
+        /// Expression-bodied local functions are normalized here to be a single expression inside
+        /// of a block statement.
```
Comment (cston) [inline src/Compilers/Core/Portable/Operations/ILocalFunctionStatement.cs]: Should the comment be `... a single expression statement inside ...`?

**Review by cston — APPROVED** [review]

**Comment by 333fred** [comment]
Ping for reviews.

**Review by mavasani — APPROVED** [review]

**Comment by AlekseyTs** [comment]
No test were affected by the change. Does this mean we have no tests for Local Functions? It would be good to have at least one otherwise how can we be sure that the affected code works at all.

**Comment by AlekseyTs** [comment]
Done with review pass.

**Comment by 333fred** [comment]
@AlekseyTs addressed.

File: src/Compilers/Core/Portable/Operations/ILocalFunctionStatement.cs (hunk 1 above)
Comment (333fred) [inline src/Compilers/Core/Portable/Operations/ILocalFunctionStatement.cs]: Updated. I changed it to be IOperation.

**Review by AlekseyTs — APPROVED** [review]
LGTM

**Comment by 333fred** [comment]
Ping @dotnet/analyzer-ioperation for a second review.

**Comment by 333fred** [comment]
Ping @dotnet/analyzer-ioperation for another review please.

**Review by CyrusNajmabadi — APPROVED** [review]

**Pull request merged**